sdk/monitor/monitor-opentelemetry/test/internal/unit/metrics/liveMetrics.test.ts

```diff
@@ -230,6 +245,13 @@ describe("#LiveMetrics", () => {
       assert.equal((documents[i].properties as any)[0].key, "customAttribute");
       assert.equal((documents[i].properties as any)[0].value, "test");
     }
+    // Ensure that requests with no URL don't throw
+    assert.strictEqual((documents[16] as Request).url, "");
```

Comment:
Would it be easier to just cast `document[16] as Request` do not duplicate that cast 4 times?